mods/modchooser/mod.yaml

```diff
@@ -4,6 +4,8 @@ Metadata:
 	Author: The OpenRA Developers
 	Hidden: true
 
+RequiresMods:
```

Comment:
Listing this everywhere imply that it's required. Making this optional would go against the whole point of the PR imo.